schema_salad/metaschema/salad.md

```diff
@@ -196,6 +197,7 @@ It is a fatal error if the document is not valid YAML.
 
 A Salad document must consist only of either a single root object or an
 array of objects.
+Each document must declare the `saladVersion` of that document. Implementations must validate against the document's declared version.
```

Comment:
This `must` here collides with the `may` clause at row 212.

Option 1: substitute this clause with

> Each document may declare the `saladVersion` of that document. Implementations may validate against the document's declared version.

Option 2: substitute row 212 with

> If a document consists of a root object, this object must contain the
field `saladVersion` and may contain the fields `$base`, `$namespaces`,
`$schemas`, and `$graph`:

Reply:
I prefer the option 2.
The option 1 makes difficult to solve #863 in the specification level.

Reply:
I totally agree to go with option 2 here.

Reply:
I fixed the row 212 to take the option 2.